### PR TITLE
Add example for all admonitions

### PR DESCRIPTION
**Pull request by froemken**

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
We want to avoid "Important" as it is rendered green (like tip and not red what you would expect), we mostly use "Attention" instead.

**Comment by froemken** [comment]
Shall I change "important" to "attention" in my PR?

**Comment by brotkrueml** [comment]
We have already a page with all admonitions: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Content/Admonitions.html, So maybe a link to that page is enough.

File: Documentation/WritingReST/CheatSheet.rst [hunk 1]
```diff
@@ -598,6 +598,8 @@ Source:
     text with a green box (as styled by our sphinx template).
     ---------------------------------------------------------------
 
+Visit :ref:`h2document:rest-admonitions` to see all allowed admonitions.
```
Comment (brotkrueml) [inline Documentation/WritingReST/CheatSheet.rst:601]:
```suggestion
Visit :ref:`h2document:rest-admonitions` to see the available admonitions.
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**